htdocs/install/page_dbsettings.php

```diff
@@ -236,19 +236,19 @@ function select_db($db_name, $link) {
 	if (empty ($error)) {
 		if (!select_db($vars ['DB_NAME'], $link)) {
 			// Database not here: try to create it
-			$result = exec_query("CREATE DATABASE `" . $vars ['DB_NAME'] . '`', $link);
+			$result = exec_query("CREATE DATABASE " . quote_sql($vars ['DB_NAME']) , $link);
 			if (!$result) {
 				$error = ERR_NO_DATABASE;
 			} else {
-				$error = sprintf(DATABASE_CREATED, $vars ['DB_NAME']);
+				$error = sprintf(DATABASE_CREATED, quote_sql($vars ['DB_NAME']));
 				$db_exist = true;
 			}
 		} else {
 			$db_exist = true;
 		}
 		if ($db_exist && $vars['DB_CHARSET']) {
 			/* Attempt to set the character set and collation to the selected */
-			$sql = "ALTER DATABASE `" . $vars ['DB_NAME'] . "` DEFAULT CHARACTER SET " . quote_sql($vars ['DB_CHARSET']) . ($vars ['DB_COLLATION']?" COLLATE " . quote_sql($vars ['DB_COLLATION']):"");
+			$sql = "ALTER DATABASE " . quote_sql($vars ['DB_NAME']) . " DEFAULT CHARACTER SET " . $vars ['DB_CHARSET'] . ($vars ['DB_COLLATION']?" COLLATE " . $vars ['DB_COLLATION']:"");
```

Comment:
You have removed here a bit too much. You didn't needed to remove allmost all `quote_sql` only quotes.